src/Form/UrlRedirectType.php

```diff
@@ -14,8 +14,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('baseUrl')
-            ->add('urlGenerator', SubmitType::class, ['label'=>'GenerateUlr'])
-        ;
+            ->add('description');
```

Comment:
chodziło o to, zeby pole description bylo dostepne tylko dla zalogowanych uzytkownikow